crates/sui-indexer-alt-jsonrpc/src/metrics/middleware.rs

```diff
@@ -138,6 +141,30 @@ where
             info!(method, elapsed_ms, "Request succeeded");
         }
 
+        if elapsed_ms > 60000.0 {
```

Comment:
similarly, hardcoding itself is not ideal -- however after looking around, I felt it even worse to let all endpoints in the middle to carry extra arg / config.

Reply:
I don't think we need to have a separate value for each endpoint. We could just keep the arg in `RpcArg` and use it in `RpcService::new` and `Context::new`.

Reply:
I'm in the same boat as @emmazzz in that I don't understand where the perceived complication is about adding a config option -- both here and for the DB. You add a config, and a default for it, and pass that in during initialisation to the metrics middleware and `PgReader`. You do not need to thread it through every place we call into the DB.